src/app/shell/personal-cabinet/applications/block-modal-window/block-modal-window.component.ts

```diff
@@ -1,9 +1,9 @@
 import { ValidationConstants } from 'src/app/shared/constants/validation';
-import { Component, Input } from '@angular/core';
+import { Component, Inject, Input } from '@angular/core';
 import { Application } from 'src/app/shared/models/application.model';
 import { FormControl, Validators } from '@angular/forms';
-import { MatDialog, MatDialogRef } from '@angular/material/dialog';
-import { ModalConfirmationText, ModalConfirmationTitle} from '../../enum/modal-confirmation';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { ModalConfirmationDescription, ModalConfirmationText, ModalConfirmationTitle, ModalConfirmationType} from '../../../../shared/enum/modal-confirmation';
 
 @Component({
   selector: 'app-block-modal-window',
```

Comment:
it shouldn't be named as block-modal, it is ReasonModal, because it could also be applied for  rejection